python/latticex/rosetta/controller/input_api.py

```diff
@@ -67,18 +67,76 @@ def private_input(party_id: int, inp):
     return _rtt.input.Input().private_input(party_id, inp)
 
 
-def private_console_input(party_id: int):
+def private_console_input(party_id: int, shape: tuple = None):
     """
     Just the same as private_input while the values will be fetched from console.
+
+    Args:
+        party_id: which party provide the private data
+        shape: only supports None, (m,), (m,n), which m, n is integer and greater 0. row first.
+
+    Usage:
+        >> rtt.private_console_input(0)
```

Comment:
```suggestion
        >> rtt.private_console_input(0)
```
    It will be better if you describe what will happen to every party. For example, if the parameter party_id is '0', does P1 and P2 will be prompted  to input?